misc/acrn-config/library/board_cfg_lib.py

```diff
@@ -39,6 +39,9 @@
     "TSN":TSN_DEVS,
 }
 
+P2SB_PASSTHRU_BOARD = ('ehl-crb-b')
+P2SB_PASSTHRU_SCENARIO = ('hybrid', 'hybrid_rt')
```

Comment:
I prefer to use vm0_pre_launch like tpm_passthru to instead specific scenario because customer might have their own scenario name.